src/Microsoft.Framework.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -218,40 +219,53 @@ public void Initialize(IEnumerable<LibraryDescription> packages, FrameworkName t
                                           IEnumerable<IPackagePathResolver> cacheResolvers,
                                           IPackage package)
         {
-            string expectedHash = null;
-
-            foreach (var resolver in cacheResolvers)
+            // REVIEW: without precomputed hash on disk, this part may hurt startup performance
```

Comment:
@davidfowl , you mentioned that SHAs should never be written to disk in theory. However, without the precomputed SHA on disk, it may hurt the performance of this part.